### ISSUE TITLE
A crash that is bound to happen.

### ISSUE THREAD

**Issue by DuanLu1112** [google/flexbox-layout#454]
![tim 20180828194318](https://user-images.githubusercontent.com/23228741/44720958-ff812600-aafa-11e8-941b-7c4a9da60396.png)
![tim 20180828194507](https://user-images.githubusercontent.com/23228741/44721021-30f9f180-aafb-11e8-88a9-52b2e9cfe32d.png)

if data source size is zero. It collapsed.
library version :1.0.0

_

**Comment by insacc** [comment]
I guess no one is really maintaining this library. The `FlexboxLayoutManager` doesn't work at all, it keeps crashing!

**Comment by thagikura** [comment]
I think it's same as #479 is it?

**Comment by DuanLu1112** [comment]
> 
> 
> I think it's same as #479 is it?
Yeah, it's the same exception as 479, but I don't know if it's the same cause as me.

**Comment by thagikura** [comment]
Do you create your own LayoutParams in the adapter of the RecyclerView?

If so, it's likely the same cause.

**Comment by DuanLu1112** [comment]
> 
> 
> Do you create your own LayoutParams in the adapter of the RecyclerView?
> 
> If so, it's likely the same cause.

No, I'm not create my own LayoutParams, I'm using it like a LinearLayoutManager.I found that the crash only happen when the data source size is 0, and it works normally when the data source size is not 0.
